web_src/js/components/RepoActionView.vue

```diff
@@ -119,13 +119,13 @@
 import {SvgIcon} from '../svg.js';
 import ActionRunStatus from './ActionRunStatus.vue';
 import {createApp} from 'vue';
-import AnsiToHTML from 'ansi-to-html';
```

Comment:
I think you are currently breaking the test for this file.

Reply:
Yes, the tests need some updates, I'm a bit lost in them currently.

Reply:
Tests are passing, still need some verification if results are ok.